src/contract/EvaluationOptionsEvaluator.ts

```diff
@@ -104,9 +104,10 @@ export class EvaluationOptionsEvaluator {
     walletBalanceUrl: () => this.rootOptions['walletBalanceUrl'],
     mineArLocalBlocks: () => this.rootOptions['mineArLocalBlocks'],
     cacheEveryNInteractions: () => this.rootOptions['cacheEveryNInteractions'],
+    useKVStorage: (foreignOptions) => foreignOptions['useKVStorage'],
```

Comment:
I believe the new parameter should be also added in if in line 122 - otherwise it will probably not work in DRE.

Reply:
Truue, nice catch, thanks